src/server/server.h

```diff
@@ -211,6 +211,7 @@ class Server {
   void SetLastRandomKeyCursor(const std::string &cursor);
 
   static int GetCachedUnixTime();
+  int GetLastBgsaveTime();
```

Comment:
```suggestion
  std::int64_t GetLastBgsaveTime();
```

Seems we can return int64 instead of int to avoid narrow conversion reported in CI.